cmd/telemetrygen/internal/metrics/metrics_types.go

```diff
@@ -0,0 +1,35 @@
+package metrics
+
+import (
+	"errors"
+)
+
+type metricType string
+
+const (
+	metricTypeAll       metricType = "all"
```

Comment:
I think it would be better to design a more general mechanism to choose how many metrics to send and customize each one. This seems like something that would take some time to design, so my preference would be to remove the `all` option for now.

Reply:
Removed the "all" option